<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="LovelyHeartSwitch" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="BoundarySwitchBase" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="BoundarySwitchBase" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="BoundarySwitchBase" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="NormalSwitch" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>326</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LovelyHeartSwitch</class>
   <extends>QWidget</extends>
   <header>lovelyheartswitch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundarySwitchBase</class>
   <extends>QWidget</extends>
   <header>boundaryswitchbase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NormalSwitch</class>
   <extends>QWidget</extends>
   <header>normalswitch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
